CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+# 3.1.0 (2019-06-07)
+
+- Added the ability to trigger responsive styles based on a parent component. [#21](https://github.com/blackbaud/skyux-lookup/pull/21)
+- Fixed the search component to properly trim whitespace when search text is applied. [#31](https://github.com/blackbaud/skyux-lookup/pull/31)
```

Comment:
How about...

Fixed the search component to trim whitespace when search text is entered.

🤷‍♀ I removed "properly" because, to me at least, that kind of implies that it trimmed whitespace incorrectly but now it's correct. I'm okay with applied instead of entered if that makes more sense for developers.

Reply:
@Blackbaud-TrevorBurch do we want to put a shoutout to Jack on the release notes for PR #31 ?

Reply:
@Blackbaud-DeniseCisneros @Blackbaud-AlexKingman updated